lib/mpl_toolkits/basemap/__init__.py

```diff
@@ -3901,6 +3901,12 @@ def drawlsmask(self,land_color="0.8",ocean_color="w",lsmask=None,
         # look for axes instance (as keyword, an instance variable
         # or from plt.gca().
         ax = kwargs.pop('ax', None) or self._check_ax()
+        # Clear saved lsmask is new lsmask is passed
+        if lsmask is not None or lsmask_lons is not None \
+                or lsmask_lats is not None:
+            # Make sure passed lsmask is not the same as cached mask
+            if lsmask != self.lsmask:
```

Comment:
Don't use equality. Use `is not` instead when comparing objects.
Also, slight typo in the comments above "lsmask is new" --> "lsmask if new"